### DISCUSSION TITLE
Docker configuration

### DISCUSSION THREAD

**Discussion by kamilpiech97** [NiclasvanEyk/initializer-for-laravel#39] [Ideas]
@NiclasvanEyk what do you think about add Docker configuration for easy development in many environments like Linux, Windows, macOS etc. . I think about downloaded files in zip archive not this repo.

**Comment by NiclasvanEyk** [comment]
Hi Kamil, could you maybe elaborate a bit? 

The zip archive already comes with Laravel Sail configured, so it contains a `docker-compose.yml` file. The concrete Dockerfiles referenced in the compose file are downloaded when installing the composer dependencies, but can be published to your project directory [as described by the Sail documentation](https://laravel.com/docs/8.x/sail#sail-customization), though this is only needed when one wants to customize the Dockerfiles, otherwise everything is prepared and intended to run via Docker.
